meta/documents/user_guide/de/wayfair_shipping.md

```diff
@@ -0,0 +1,121 @@
+
+1. Gehen Sie auf der Hauptseite von Plentymarkets zu `Einrichtung` >> `Aufträge` >> `Versand` >> `Optionen`.
+
+  ![shipping settings](../../../images/de/menu_order_shipping_settings.png)
+
+2. Wählen Sie die Registerkarte `Versanddienstleister`.
+
+  ![shipping service providers](../../../images/de/shipping/shipping_service_providers_without_wayfair.png)
+
+3. Klicken Sie auf die Schaltfläche `+ Neu` ![new button](../../../images/de/button_plus_new.png), um der Tabelle eine neue Zeile hinzuzufügen.
+
+4. Füllen Sie in der neu erstellten Zeile das Feld `Name` aus (der Einfachheit halber empfehlen wir **"WayfairShipping"** für alle Namen).
+
+5. Wählen Sie im Feld `Versanddienstleister` für die Zeile die Option `WayfairShipping` aus. Dies entspricht dem Typ, der vom Wayfair-Plugin bereitgestellt wird. **Alle anderen Felder in der Zeile können leer bleiben.**
+
+6. Stellen Sie sicher, dass die neue Zeile folgendermaßen aussieht:
+    ![shipping service providers](../../../images/de/shipping/shipping_service_providers_with_wayfair.png)
+
+7. Klicken Sie auf "Speichern" ![save button](../../../images/de/button_save1.png).
+
+## 2. Versandprofil erstellen
+Jede Bestellung verfügt über ein Versandprofil, das für die Zusammenarbeit mit dem Versanddienstleister verwendet wird, um Artikel aus einer Bestellung zu versenden. Führen Sie diese Schritte aus, um das Wayfair-spezifische Versandprofil einzurichten.
+
+1. Gehen Sie auf der Hauptseite von Plentymarkets zu `Einrichtung` >> `Aufträge` >> `Versand` >> `Optionen`.
+
+    ![shipping settings](../../../images/de/menu_order_shipping_settings.png)
+
+2. Wählen Sie die Registerkarte `Versandprofile`.
+
+    ![shipping profiles](../../../images/de/shipping/shipping_profiles_without_wayfair.png)
+
+3. Wenn bereits eine "Wayfair"-Zeile vorhanden ist, klicken Sie auf diese Zeile. Andernfalls klicken Sie auf die Schaltfläche `Neu`![new button](../../../images/de/button_plus_new.png). Die Einstellungen des Versandprofils werden angezeigt.
+
+4. Wählen Sie Ihren Wayfair-Versanddienstleister aus dem bereitgestellten Menü aus (es wird wahrscheinlich **"WayfairShipping"** sein, wenn Sie [den vorgeschlagenen Versanddienstleister](#1-erstellen-des-versanddienstleisters) erstellt haben.
+
+    ![select wayfairshipping provider](../../../images/de/shipping/select_wayfairshipping_provider.png)
+
+
+5. Füllen Sie die Felder aus, welche sich auf `Name` beziehen. (der Einfachheit halber empfehlen wir **"WayfairShipping"** für alle Namen).
+
+6. Stellen Sie die Sprache über das Menü rechts neben dem Feld `Name` ein.
+
+7. Wählen Sie im Feld `Markierung` das Symbol für Wayfair-Bestellungen aus (`6` und `126` sind gute Optionen, da sie das Wayfair-Farbschema widerspiegeln).
+
+8. Ändern Sie im Feld `Priorität` optional die Priorität (wir empfehlen, sie als Standardpriorität `höchste` zu belassen).
+
+9. Stellen Sie im Feld Clients (Stores) sicher, dass Sie alle Stores auswählen, die für Wayfair eingerichtet werden können.

```

Comment:
Verify proper German translation from Falco.
Should say “In the Clients (stores) field, make sure to select all stores that may be set up for Wayfair.”